packages/components/addon/components/hds/side-nav/list/title.hbs

```diff
@@ -4,5 +4,7 @@
 }}
 
 <Hds::SideNav::List::Item>
-  <div class="hds-side-nav__list-title hds-typography-body-100 hds-font-weight-semibold" ...attributes>{{~yield~}}</div>
+  <div class="hds-side-nav__list-title" ...attributes>
+    <span class="hds-side-nav__list-title-content hds-typography-body-100 hds-font-weight-semibold">{{~yield~}}</span>
```

Comment:
I don't think we need an extra `<span>` here: I did a quick test and if we remove the `display: flex` from the `hds-side-nav__list-title-content` (not needed, I'm not sure why it was added - probably by me - in the first place) and apply a `overflow-wrap: break-word;` to it, then it works as expected

Reply:
Ok thanks, I wasn't sure why you added display: flex but thought maybe it was to make it work with some other kind of content. I can remove & clean up.